modules/javafx.graphics/src/main/java/javafx/scene/image/Image.java

```diff
@@ -78,19 +79,28 @@
  * memory consumed by the image). The application can specify the quality of
  * filtering used when scaling, and whether or not to preserve the original
  * image's aspect ratio.
- * </p>
  *
- * <p>
- * All URLs supported by {@link URL} can be passed to the constructor.
- * If the passed string is not a valid URL, but a path instead, the Image is
- * searched on the classpath in that case.
- * </p>
+ * <p>If a URL string is passed to a constructor, it be any of the following:
+ * <ol>
+ *     <li>the name of a resource that can be resolved by the context
+ *         {@link ClassLoader} for this thread
+ *     <li>a file path that can be resolved by {@link java.io.File}
+ *     <li>a URL that can be resolved by {@link java.net.URL} and for
+ *         which a protocol handler exists
+ * </ol>
+ *
+ * <p>The RFC 2397 "data" scheme for URLs is supported in addition to
+ * the protocol handlers that are registered for the application.
+ *
+ * <p>If a URL uses the "data" scheme, the data must be base64-encoded
```

Comment:
I think these two sentences should be in the same paragraph.

Reply:
Ok.